.github/workflows/handler-test.yml

```diff
@@ -760,4 +787,4 @@ jobs:
 
             ${{ format('- {0} Run k6 Smoke Test', steps.run-smoke-test.outcome == 'success' && ':white_check_mark:' || ':x:') }}
 
-            ${{ github.event.client_payload.slash_command.args.named.destroy != 'false' && format('- {0} Terraform Destroy', steps.destroy.outcome == 'success' && ':white_check_mark:' || ':x:') || '' }}
+            ${{ github.event.client_payload.slash_command.args.named.destroy != 'false' && format('- {0} Terraform Destroy', steps.destroy.outcome == 'success' && ':white_check_mark:' || ':x:') || '' }}
```

Comment:
Added similar section for destroy-handler.yml as well.
Since only empty workspaces can be destroyed it makes sense to use same `/destroy` command to destroy resources  and delete the transient workspaces.